webcomponents/tool/src/main/java/org/sakaiproject/webcomponents/permissions/PermissionsEntityProvider.java

```diff
@@ -112,42 +114,60 @@ public ActionReturn handleGet(EntityView view, Map<String, Object> params) {
             }
         }
 
+        String overrideReference = (String) params.get("overrideRef");
+        AuthzGroup overrideAuthzGroup = null;
+        if (StringUtils.isNotBlank(overrideReference)) {
+            try {
+                overrideAuthzGroup = authzGroupService.getAuthzGroup(overrideReference);
+            } catch (GroupNotDefinedException e) {
+                throw new IllegalArgumentException("No realm defined for override ref " + overrideReference);
```

Comment:
So if there is not Authz Group for overrideReference, can we just log it and continue?

Reply:
Probably.

Reply:
I'm not too sure about just logging it, tbh. If a tool specifies an override authzgroup and that group doesn't exist, I think that should be an exception. I might add some error handling in the web component for that case.